### PR TITLE
Fix send message

### PR DESCRIPTION
**Pull request by remk**

### CODE REVIEW

**Review by blackheaven — APPROVED** [review]

**Review by rouazana — APPROVED** [review]

File: src/main/scala-2.12/org/apache/james/gatling/imap/scenari/ImapFullScenario.scala [hunk 1]
```diff
@@ -25,7 +25,7 @@ class ImapFullScenario {
 
   val readLastEmail = exec(imap("list").list("", "*").check(ok, hasFolder("INBOX")))
     .exec(imap("select").select("INBOX").check(ok))
-    .exec(imap("fetch").fetch(MessageRanges(Last()), AttributeList("BODY[HEADER]", "UID", "BODY[TEXT]")).check(ok))
+    .exec(imap("fetch").fetch(MessageRanges(Last()), AttributeList("UID", "BODYSTRUCTURE")).check(ok))
```
Comment (mbaechler) [inline src/main/scala-2.12/org/apache/james/gatling/imap/scenari/ImapFullScenario.scala:31]: it deserves a comment IMO

**Review by mbaechler — APPROVED** [review]

File: src/main/scala-2.12/org/apache/james/gatling/imap/scenari/ImapFullScenario.scala [hunk 2]
```diff
@@ -25,6 +25,9 @@ class ImapFullScenario {
 
   val readLastEmail = exec(imap("list").list("", "*").check(ok, hasFolder("INBOX")))
     .exec(imap("select").select("INBOX").check(ok))
+    // The imapnio library fail to parse the attributes BODY[TEXT] and BODY[HEADER]
+    // so for the time being we will only fetch the BODYSTRUCTURE
+    // the corresponding issue can be found at :  https://ci.linagora.com/linagora/lgs/openpaas/james/issues/1243
```
Comment (rouazana) [inline src/main/scala-2.12/org/apache/james/gatling/imap/scenari/ImapFullScenario.scala]: agree, but ci.linagora.com is not available for external users, so please create a github issue and reference it inside

**Review by Arsnael — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Comment by remk** [comment]
Fix https://github.com/linagora/james-gatling/issues/54

**Review by blackheaven — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Review by rouazana — APPROVED** [review]

**Review by mbaechler — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Comment by rouazana** [comment]
merged

**Pull request closed without merging**